pkg/converter/source/virtualmachine_converter.go

```diff
@@ -62,22 +62,22 @@ func (v VMConverter) Start() {
 	}
 }
 
-func (v VMConverter) processEvent(vm *VirtualMachineSource, failedUpdates map[string]retryRecord, isRetry bool, isAgent bool) {
+func (v VMConverter) processEvent(vm *VirtualMachineSource, failedUpdates map[string]retryRecord, isRetry bool, isExternalnode bool) {
 	var err error
 	var fetchKey client.ObjectKey
-	log := v.Log.WithName("processEvent")
 
-	if isAgent {
+	if isExternalnode {
 		fetchKey = target.GetExternalNodeKeyFromSource(vm)
 	} else {
 		fetchKey = target.GetExternalEntityKeyFromSource(vm)
 	}
-	log.Info("Received event", "Key", fetchKey, "Agented", isAgent)
+	v.Log.Info("Received event", "Key", fetchKey, "Agented", isExternalnode)
```

Comment:
Why Agented here, and isExternalNode later.. be consistent.. Probably just use ExternalNode?